rfc/2022/2022-08-23_CIT_hostnames.md

```diff
@@ -0,0 +1,227 @@
+  - Once implemented, the new Apigee gateway will continue to honor legacy/mobile app requests to `api.va.gov/mobile/*`.
+  - Once implemented, the new Apigee gateway will continue to honor requests to `api.va.gov/vanotify` and route them to the  VANotify service until that team agrees to retire the endpoint.
+
+
+#### **Infrastructure Team**
+  - Removes api.va.gov references from the revProxy.
+  - **Note:** At this time only the Dev website is in EKS. As other environments migrate to EKS, routing configurations will need to be updated and release timing will need to be re-evaluated.
+
+    Reference: [Enabling new *platform-api.va.gov domains in EKS](https://vfs.atlassian.net/wiki/spaces/CLOUD/pages/2263351375).
+  - Schedule annual TLS/SSL cert renewals for all platform-api subdomains.
+
+
+#### **Documentation**
+  - The Platform Content Team coordinates with contributing teams to edit documentation to use the *platform-api.va.gov hostname.
+  - The Console UI Team coordinates with contributing teams to edit documentation to use the *platform-api.va.gov hostname.
+
+
+## **Risks**
+  - **Lack of parity in environments and deployment processes:** Consideration should be given to completing the migration of all environments to EKS prior to rolling out *platform-api.va.gov.
+    - Reduces complexity in Release Plan processes.
+    - Reduces complexity in Post-Release issue resolution.
+  - **Staging downtime:** A window of time will be required to allow teams to test and mitigate any unforeseen issues. Authentication flows and day-to-day development tasks may be interrupted.
+  - **Production downtime:** There may be a short period of downtime or instability in authentication flows.
+  - **Unidentified issues:** Issues that are not identified in the Staging release may affect Production.
+
+
+## **Appendix**
+  - [Forward Proxy Addresses](https://vfs.atlassian.net/wiki/spaces/CLOUD/pages/2263449831/Guide+API+Hostname+Status) (Confluence)
+  - [Hostname Specifications](https://vfs.atlassian.net/wiki/spaces/CLOUD/pages/2283339976) (Confluence)
+  - [Reroute vets-api upstream requests to Lighthouse](https://vfs.atlassian.net/wiki/spaces/ECP/pages/2220785993/2022-06-06+Reroute+vets-api+upstream+requests+to+Lighthouse) (ADR/Confluence)
+  - [Datadog Tests](https://vagov.ddog-gov.com/synthetics/tests?query=tag%3A%28%22team%3Acit%22%29) (Datadog Link)
+  - [Remapped vets-api settings](https://docs.google.com/spreadsheets/d/111t6f4V3eCVkaKoBIxXi54_HlkPT-54qKShfRjl-iMs/edit#gid=2101280441) (Google Sheet)
+  - [Enabling new *platform-api.va.gov domains in EKS](https://vfs.atlassian.net/wiki/spaces/CLOUD/pages/2263351375) (Confluence)
+  - [New Domain to Serve vets-api](https://vfs.atlassian.net/wiki/spaces/ECP/pages/2221572288/2022-06-06+New+domain+to+serve+vets-api) (ADR/Confluence)
```

Comment:
Additional background & references:
- https://vfs.atlassian.net/wiki/spaces/ECP/pages/1907523684/Explainer+-+VA.gov-Lighthouse+Overlap
- https://github.com/department-of-veterans-affairs/va.gov-team/discussions/39068

Reply:
Thanks, I've added those links to the Appendix.